CHANGES

```diff
@@ -2,6 +2,8 @@
 - [BREAKING CHANGE] GWT: Updated to 2.10.0. `com.google.jsinterop:jsinterop-annotations:2.0.2:sources` must be added as a dependency to your html project dependencies.
 - Fixed GlyphLayout for fixed width glyph offsets at the start and end of lines.
 - Fixed scene2d.ui layout for fractional positions and sizes.
+- libGDX is now built using Java 17 due to Gradle 8 requirements.
```

Comment:
I don't think I see this change in the list? Did a commit get lost?

Reply:
heheh, I've added here but it should've been added to https://github.com/libgdx/libgdx/pull/7246